packages/sagas/.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: false
+node_js:
+  - '4'
+  - '6'
+before_script:
+  - npm prune
+  - npm update
+after_success:
+  - 'curl -Lo travis_after_all.py https://git.io/travis_after_all'
+  - python travis_after_all.py
+  - export $(cat .to_export_back) &> /dev/null
+  - npm run semantic-release
+branches:
+  except:
+    - /^v\d+\.\d+\.\d+$/
```

Comment:
this file is not needed

Reply:
also packages/sagas/index.js is not needed

Reply:
It does not hurt and will be needed as soon as a second saga will be added, plus it keeps homogeneity with other packages.